### PR TITLE
Style

### PR DESCRIPTION
**Pull request by maubertw**
### Assignee Tasks
***add product category link is displaying despite not being logged it, so I have it commented out in this branch, as it was disruptive to the flow.  I will submit an issue for fixing that feature*****

****also, I want to adjust the look of the add to cart area and the logo on the nav bar, but I will adjust on final tweak when all components have minimum styling******

- [ x] added product cards
- [ x] adjust quantity and add to cart from all products
-[x]single product page style

connect to #77

### CODE REVIEW

File: client/components/product/number-picker-wrapper.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+
+
+  render(){
+    const quantity = this.state.quantity
+    return(
+      <div>
+      <Segment.Group horizontal >
+        <p>qantityty:</p>
```
Comment (chawes13) [inline client/components/product/number-picker-wrapper.js]: small typo

File: client/components/product/number-picker-wrapper.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+class NumberPickerWrapper extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      quantity: 1
+    }
+  }
+
+  updateQuantity = (event) => {
```
Comment (chawes13) [inline client/components/product/number-picker-wrapper.js]: are you using this anymore?

File: client/components/products.js [hunk 3]
```diff
@@ -71,43 +69,34 @@ class Products extends React.Component {
     const products = this.state.showProducts;
     return (
       <Container >
-        <h3>ALL PRODUCTS</h3>
-        <Link to="/admin/products/add">Add Product</Link>
```
Comment (chawes13) [inline client/components/products.js:75]: where should this be moved to?

**Review by chawes13 — CHANGES_REQUESTED** [review]
Fix small typo and delete method (if not using anymore)

**Review by chawes13 — APPROVED** [review]

File: client/components/products.js (hunk 3 above)
Comment (maubertw) [reply client/components/products.js:75]: dashboard!

**Pull request merged**